lib/pbench/test/unit/server/test_users_api.py

```diff
@@ -0,0 +1,48 @@
+from http import HTTPStatus
+
+from pbench.test.unit.server import DRB_USER_ID
+from pbench.test.unit.server.headertypes import HeaderTypes
+
+
+class Testuser:
+    def test_get_user(self, client, server_config, build_auth_header):
+        header_param = build_auth_header["header_param"]
+        expected_status = HTTPStatus.OK
+        if not HeaderTypes.is_valid(header_param):
+            expected_status = HTTPStatus.UNAUTHORIZED
+
+        response = client.get(
+            f"{server_config.rest_uri}/user/drb",
+            headers=build_auth_header["header"],
+        )
+        assert response.status_code == expected_status
+        if header_param is HeaderTypes.VALID_ADMIN:
+            assert response.json == {
+                "username": "drb",
+                "id": int(DRB_USER_ID),
+                "oidc_id": DRB_USER_ID,
+                "roles": [],
+            }
+        elif header_param is HeaderTypes.VALID:
+            assert response.json == {
+                "username": "drb",
```

Comment:
... and of course the user `id` is also inconsistent with this case where we don't have one.

Reply:
The requesting user is not an admin here so the API doesn't return the internal row id here.

Reply:
I was arguing about the "why" but simply the "what" ... the inconsistency serves no purpose.